__tests__/integration/mirador/plugins/order.test.js

```diff
@@ -0,0 +1,15 @@
+
+describe('try to apply 2 add plugins and 2 wrap plugins to <WorkspaceControlPanelButtons>', () => {
```

Comment:
Maybe the term `nesting` or `wrapping` would be more suitable, instead of `order`